Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -457,14 +456,18 @@ - (void) URLSession: (NSURLSession *)session
 
 - (void)URLSession:(NSURLSession *)session readClosedForStreamTask:(NSURLSessionStreamTask *)task
 {
-    [self streamClosed: NO];
+    if (task == _task)
```

Comment:
It'd be cleaner to move this test (and the one in the next method) into -streamClosed:, since we're trying to share the logic for both the read and write streams.